### PR TITLE
Feature/i44

### PR DESCRIPTION
**Pull request by TheBlackArroVV**
image
image

### CODE REVIEW

File: app/views/contacts/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<%= render 'form', contact: @contact%>
```
Comment (AZatsepa) [inline app/views/contacts/edit.html.erb]: Add a whitespace before @contact

File: app/views/organizations/show.html.erb [hunk 2]
```diff
@@ -3,4 +3,13 @@
 <%= @organization.description %><br/>
 <%= link_to 'Back', organizations_path %>
 <%= link_to 'Update', edit_organization_path(@organization) %>
-<%= button_to 'Destroy', organization_path(@organization), method: :delete %>
\ No newline at end of file
+<%= button_to 'Destroy', organization_path(@organization), method: :delete %>
```
Comment (AZatsepa) [inline app/views/organizations/show.html.erb]: I18n

File: app/views/contacts/edit.html.erb (hunk 1 above)
Comment (TheBlackArroVV) [reply app/views/contacts/edit.html.erb]: *after

**Comment by davinciRor** [comment]
We added semanticUI. Please use it.

File: app/views/organizations/show.html.erb [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
 <%= @organization.description %><br/>
-<%= link_to 'Back', organizations_path %>
-<%= link_to 'Update', edit_organization_path(@organization) %>
-<%= button_to 'Destroy', organization_path(@organization), method: :delete %>
\ No newline at end of file
+<%= link_to t('crud.actions.back'), organizations_path %>
+<%= link_to t('crud.actions.update'), edit_organization_path(@organization) %>
+<%= button_to t('crud.actions.destroy'), organization_path(@organization), method: :delete %>
+
+<p>Contacts</p>
```
Comment (davinciRor) [inline app/views/organizations/show.html.erb]: I18n

**Comment by davinciRor** [comment]
Looks good for me for now. But you should add beautiful UI.

File: config/locales/uk.yml [hunk 4]
```diff
@@ -4,9 +4,16 @@ uk:
   account_has_been_locked: Ваш обліковий запис було заблоковано через надто високу кількість неправильних спроб увійти.
   activerecord:
     attributes:
+      contacts:
```
Comment (AZatsepa) [inline config/locales/uk.yml]: Should be `contact` for automatic usage in forms

File: config/environments/test.rb [hunk 5]
```diff
@@ -47,4 +47,5 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.i18n.locale = :en
```
Comment (AZatsepa) [inline config/environments/test.rb]: We`ve already set a default locale in the config/initializers/locale.rb

**Pull request merged**